src/templates/package.json.squirrelly

```diff
@@ -40,7 +40,7 @@
     "@babel/preset-env": "^7.13.12",
     "@babel/runtime": "^7.13.10",
     "@babel/runtime-corejs2": "^7.13.10",
-    "@oliveai/ldk": "3.1.0",
+    "@oliveai/ldk": "^3.2.0",
```

Comment:
I went ahead and used the looser ^ version specifier. @BrettMugglinOlive [is confident](https://crosschx.slack.com/archives/C01JFU3LQF4/p1630619984024400?thread_ts=1630613173.024000&cid=C01JFU3LQF4). 😄